tests/gui/test_commands.py

```diff
@@ -391,11 +392,18 @@ def assert_skeletons_match(new_skeleton: Skeleton, skeleton: Skeleton):
 
     def OpenSkeleton_ask(context: CommandContext, params: dict) -> bool:
         """Implement `OpenSkeleton.ask` without GUI elements."""
-
-        # Original function opens FileDialog here
-        filename = params["filename_in"]
-
+        template = (
+            context.app.currentText
+        )  # Original function uses `QComboBox.currentText()`
+        if template == "Custom":
+            # Original function opens FileDialog here
+            print("Loading from file dialog")
+            filename = params["filename_in"]
+        else:
+            print("Loading from template")
+            filename = get_package_file(f"sleap/skeletons/{template}.json")
         if len(filename) == 0:
+            print("Whoops, filename 0.")
```

Comment:
Remove print statements
```suggestion
            filename = params["filename_in"]
        else:
            filename = get_package_file(f"sleap/skeletons/{template}.json")
        if len(filename) == 0:
```